### PR TITLE
net-im/bitlbee-facebook: new ebuild

### PR DESCRIPTION
**Pull request by arkamar**
Gentoo-Bug: https://bugs.gentoo.org/550184
Package-Manager: Portage-2.3.13, Repoman-2.3.3

### CODE REVIEW

File: net-im/bitlbee-facebook/bitlbee-facebook-1.1.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+inherit eutils autotools
+
+DESCRIPTION="Facebook protocol plugin for BitlBee"
+HOMEPAGE="https://github.com/bitlbee/bitlbee-facebook"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="debug"
+
+if [ "${PV}" = "9999" ] ; then
```
Comment (titanofold) [inline net-im/bitlbee-facebook/bitlbee-facebook-1.1.2.ebuild]: It'd be better to not have a conditional SRC. We should really limit how much code is run in the global space.

File: net-im/bitlbee-facebook/bitlbee-facebook-1.1.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		$(use_enable debug) \
+		--enable-minimal-flags
```
Comment (titanofold) [inline net-im/bitlbee-facebook/bitlbee-facebook-1.1.2.ebuild]: I can't find what effect this is supposed to have.

File: net-im/bitlbee-facebook/bitlbee-facebook-9999.ebuild [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+inherit eutils autotools
+
+DESCRIPTION="Facebook protocol plugin for BitlBee"
+HOMEPAGE="https://github.com/bitlbee/bitlbee-facebook"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="debug"
+
+if [ "${PV}" = "9999" ] ; then
```
Comment (titanofold) [inline net-im/bitlbee-facebook/bitlbee-facebook-9999.ebuild]: Same as the last. Eliminate the conditional.

File: net-im/bitlbee-facebook/bitlbee-facebook-1.1.2.ebuild (hunk 1 above)
Comment (SoapGentoo) [reply net-im/bitlbee-facebook/bitlbee-facebook-1.1.2.ebuild]: This fine and absolutely standard in the Gentoo tree, no reason to change.
Still, use bash and not POSIX: `if [[ ${PV} == *9999 ]] ; then`

File: net-im/bitlbee-facebook/bitlbee-facebook-1.1.2.ebuild [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+src_install() {
+	default
+	prune_libtool_files --all
```
Comment (SoapGentoo) [inline net-im/bitlbee-facebook/bitlbee-facebook-1.1.2.ebuild]: https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Handling_Libtool_Archives

	find "${D}" -name '*.la' -delete || die

and then remove inheriting `eutils`

File: net-im/bitlbee-facebook/bitlbee-facebook-9999.ebuild (hunk 3 above)
Comment (SoapGentoo) [reply net-im/bitlbee-facebook/bitlbee-facebook-9999.ebuild]: Leave it, standard Gentoo idiom

File: net-im/bitlbee-facebook/bitlbee-facebook-1.1.2.ebuild (hunk 4 above)
Comment (arkamar) [reply net-im/bitlbee-facebook/bitlbee-facebook-1.1.2.ebuild]: Ok, thanks.

File: net-im/bitlbee-facebook/bitlbee-facebook-1.1.2.ebuild (hunk 2 above)
Comment (arkamar) [reply net-im/bitlbee-facebook/bitlbee-facebook-1.1.2.ebuild]: Well, it seems that it does not heave any effect anymore and --enable-debug also. I remove it.

**Comment by SoapGentoo** [comment]
@arkamar looks good, please squash all commits into one and use the `Bug:` style (or `Closes:` if it fixes the bug).
Squashing commits: https://wiki.gentoo.org/wiki/Gentoo_GitHub#Rebasing_commits

**Comment by SoapGentoo** [comment]
Merged

**Pull request closed without merging**